src/components/autocomplete/js/autocompleteController.js

```diff
@@ -208,6 +208,17 @@ function MdAutocompleteCtrl ($scope, $element, $mdUtil, $mdConstant, $mdTheming,
   }
 
   /**
+   * Find the target for the wrap element which determines menu width
+   * @returns {*}
+   */
+  function getWrapTarget() {
+    for (var element = $element; element.length; element = element.parent()) {
```

Comment:
That's not a bad idea; although I am a bit confused as to how this doesn't already affect the width of the item. Are positioning and width somehow separate?

(I didn't actually know `md-autocomplete-snap` existed since it's not in the docs)